### DISCUSSION TITLE
Jellyfin support along with plex

### DISCUSSION THREAD

**Discussion by hackmonker** [umbrellaplug/umbrellaplug.github.io#327] [General]
Hey. First of all wanted to say love our addon. It has been going real smooth for me. Thanks for all your hardwork.

I know there is the plex scrapping integration but was wondering if you were also interested in adding a jellyfin integration to. Jellyfin is a plex opensource alternative and has become very popular recently as well. So would love to see it also be a part of the addon. You might be already aware of it but putting down the link just in case. Cheers

https://jellyfin.org/

**Comment by umbrellaplug** [comment]
I have no plans to add jellyfin support at this time.

**Comment by hackmonker** [comment]
No problem. Just wanted to try and ask. Hope it will be considered in future if you gain interest or get time to invest in it. Thanks
